packages/compass-e2e-tests/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "@mongodb-js/tsconfig-compass/tsconfig.common.json",
   "compilerOptions": {
     "noEmit": true,
-    "moduleResolution": "node10"
+    "module": "nodenext",
+    "moduleResolution": "nodenext"
```

Comment:
Was easier to keep everything else on node16 and only switch e2e to nodenext, I think in theory we might want to use this module config option everywhere, but I didn't want to deal with implications in this PR